enterprise/frontend/.eslintrc

```diff
@@ -34,4 +34,13 @@
     ],
     "ttag/no-module-declaration": 2,
   },
+  "overrides": [
```

Comment:
moved from `enterprise/frontend/src/embedding-sdk/.eslintrc` because now we have a new directory `embedding` and I also don't want these rules to apply to storybook stories under this new directory.